src/Adapters/Auth/index.js

```diff
@@ -104,12 +120,12 @@ module.exports = function(authOptions = {}, enableAnonymousUsers = true) {
     } = loadAuthAdapter(provider, authOptions);
 
     return authDataValidator(adapter, appIds, providerOptions);
-  }
+  };
 
   return Object.freeze({
     getValidatorForProvider,
     setEnableAnonymousUsers
-  })
-}
+  });
+};
```

Comment:
Not needed for this PR, let's kept it focused on what you're actually introducing.